fudy/NoteScreen/Component/FramelessWindow.qml

```diff
@@ -9,13 +9,16 @@ ApplicationWindow {
 
 	property string headerText: ""
 	property alias contentComponent: contentLoader.sourceComponent
+	property alias loaderItem: contentLoader.item
 	property int index
```

Comment:
Index what?

Reply:
Index of the task card that we want to hide. I will update the name to make it more understandable.